### PR TITLE
Refactor marquees

### PR DESCRIPTION
**Pull request by mirhamasala**
- Use a `for` loop for repetitive marquee content

### CODE REVIEW

File: priv/site/_includes/tickets/marquee.slime [hunk 1]
```diff
@@ -1,32 +1,9 @@
...
-
-  a href="#"
-    span Buy Tickets Here
-    <svg width="56" height="56" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <rect width="56" height="56" fill="#E5E5E5"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M44.8681 27.6127L33.9229 14.7288L36.4419 12.5632L49.2268 27.6127L36.4419 42.6622L33.9229 40.4966L44.8681 27.6127Z" fill="black"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M6.56641 25.946L46.6983 25.946L46.6983 29.2794L6.56641 29.2794L6.56641 25.946Z" fill="black"/>
-    </svg>
+  = for x <- 1..4 do
+    a href="#"
```
Comment (frm) [inline priv/site/_includes/tickets/marquee.slime:3]: Oof, this actually should've been a `tito_url()`. Can you replace this with a call to `link/3`?

File: priv/site/_includes/info.slime [hunk 2]
```diff
@@ -75,15 +75,6 @@
...
-      span Soon on a screen near you
-      span Join us
-      span Soon on a screen near you
-      span Join us
-      span Soon on a screen near you
-      span Join us
-      span Soon on a screen near you
-      span Join us
-      span Soon on a screen near you
+      = for x <- 1..6 do
```
Comment (frm) [inline priv/site/_includes/info.slime:78]: Same as previous comment

File: priv/site/_includes/tickets/marquee.slime [hunk 3]
```diff
@@ -1,32 +1,9 @@
...
-    </svg>
-
-  a href="#"
-    span Buy Tickets Here
-    <svg width="56" height="56" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <rect width="56" height="56" fill="#E5E5E5"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M44.8681 27.6127L33.9229 14.7288L36.4419 12.5632L49.2268 27.6127L36.4419 42.6622L33.9229 40.4966L44.8681 27.6127Z" fill="black"/>
-      <path fill-rule="evenodd" clip-rule="evenodd" d="M6.56641 25.946L46.6983 25.946L46.6983 29.2794L6.56641 29.2794L6.56641 25.946Z" fill="black"/>
-    </svg>
+  = for x <- 1..4 do
```
Comment (frm) [inline priv/site/_includes/tickets/marquee.slime:2]: Same as previous comment.

File: priv/site/_includes/hero.slime [hunk 4]
```diff
@@ -25,15 +25,10 @@ header.Banner
...
-      span All proceeds are for charity
-      span This is a carbon-neutral event
-      span 1 attendee = 1 tree
-      span All proceeds are for charity
-      span This is a carbon-neutral event
-      span 1 attendee = 1 tree
-      span All proceeds are for charity
-      span This is a carbon-neutral event
-      span 1 attendee = 1 tree
+      = for x <- 1..3 do
```
Comment (frm) [inline priv/site/_includes/hero.slime:28]: Looks good but this will give out a warning due to `x` being unused. We can do `for _ <- 1..3` instead.

**Review by frm — CHANGES_REQUESTED** [review]
Looking good! 👍 Just a few minor issues

File: priv/site/_includes/tickets/marquee.slime (hunk 1 above)
Comment (mirhamasala) [reply priv/site/_includes/tickets/marquee.slime]: I've changed all to `= link @env, to: tito_url() do`.

Let me know if I missed something–not sure what you meant with "Can you replace this with a call to `link/3`?"

Comment (frm) [reply priv/site/_includes/tickets/marquee.slime]: That's perfect, it was precisely it 👍 

`function/arity` is the notation to describe functions in Elixir. So `link/3` means "function named `link` that receives 3 arguments". Just a fancy way to distinguish between different functions and function branches.

**Review by frm — APPROVED** [review]

**Comment by frm** [comment]
PS: when merging can you use "Squash and merge"?

File: priv/site/_includes/tickets/marquee.slime (hunk 1 above)
Comment (mirhamasala) [reply priv/site/_includes/tickets/marquee.slime]: Ah, nice. Thank you.

**Pull request merged**